bitnami/mxnet/templates/distributed/scheduler/deployment.yaml

```diff
@@ -0,0 +1,215 @@
+          {{- end }}
+          {{- if .Values.scheduler.lifecycleHooks }}
+          lifecycle: {{- include "common.tplvalues.render" (dict "value" .Values.scheduler.lifecycleHooks "context" $) | nindent 12 }}
+          {{- end }}
+          env:
+            - name: DMLC_ROLE
+              value: "scheduler"
+            - name: DMLC_NUM_WORKER
+              value: {{ .Values.worker.replicaCount | quote }}
+            - name: DMLC_NUM_SERVER
+              value: {{ .Values.server.replicaCount | quote }}
+            - name: DMLC_PS_ROOT_URI
+              valueFrom:
+                fieldRef:
+                  fieldPath: status.podIP
+            - name: DMLC_PS_ROOT_PORT
+              value: {{ .Values.scheduler.containerPorts.mxnet | quote }}
+            {{- if .Values.extraEnvVars }}
+            {{- include "common.tplvalues.render" (dict "value" .Values.extraEnvVars "context" $) | nindent 12 }}
+            {{- end }}
+            {{- if .Values.scheduler.extraEnvVars }}
+            {{- include "common.tplvalues.render" (dict "value" .Values.scheduler.extraEnvVars "context" $) | nindent 12 }}
+            {{- end }}
+          envFrom:
+            {{- if .Values.extraEnvVarsCM }}
+            - configMapRef:
+                name: {{ include "common.tplvalues.render" (dict "value" .Values.extraEnvVarsCM "context" $) }}
+            {{- end }}
+            {{- if .Values.extraEnvVarsSecret }}
+            - secretRef:
+                name: {{ include "common.tplvalues.render" (dict "value" .Values.extraEnvVarsSecret "context" $) }}
+            {{- end }}
+            {{- if .Values.scheduler.extraEnvVarsCM }}
+            - configMapRef:
+                name: {{ include "common.tplvalues.render" (dict "value" .Values.scheduler.extraEnvVarsCM "context" $) }}
+            {{- end }}
+            {{- if .Values.scheduler.extraEnvVarsSecret }}
+            - secretRef:
+                name: {{ include "common.tplvalues.render" (dict "value" .Values.scheduler.extraEnvVarsSecret "context" $) }}
+            {{- end }}
+          ports:
+            - name: mxnet
+              containerPort: {{ .Values.scheduler.containerPorts.mxnet }}
+          {{- if not .Values.diagnosticMode.enabled }}
+          {{- if .Values.scheduler.livenessProbe.enabled }}
+          livenessProbe: {{- include "common.tplvalues.render" (dict "value" (omit .Values.scheduler.livenessProbe "enabled") "context" $) | nindent 12 }}
+            tcpSocket:
+              port: mxnet
+          {{- else if .Values.scheduler.customLivenessProbe }}
+          livenessProbe: {{- include "common.tplvalues.render" (dict "value" .Values.scheduler.customLivenessProbe "context" $) | nindent 12 }}
+          {{- end }}
+          {{- if .Values.scheduler.readinessProbe.enabled }}
+          readinessProbe: {{- include "common.tplvalues.render" (dict "value" (omit .Values.scheduler.readinessProbe "enabled") "context" $) | nindent 12 }}
```

Comment:
Not sure if this is correct, but the scheduler entrypoint by default is `sleep infinity` and there is nothing listening in `mxnet` port so the container is never ready